src/diffpy/srfit/pdf/characteristicfunctions.py

```diff
@@ -415,6 +415,13 @@ def __call__(self, r):
         #
         # We also have to make a q-spacing small enough to compute out to at
         # least the size of the signal.
+
+        # As of release 3.2.0, these are not working but we hope to have
```

Comment:
we would like this comment to be the `NotImplemented` error message.  No user will see this comment.

I am not sure how it is used, but do we also want this behavior in `__init__()` constructor?  Or is there some reason we may want it instantiated but not called?